script/Ciel/Day 13/13_01_02_CIEL13_6_1.txt

```diff
@@ -306,7 +325,7 @@ Please, God. Make her happy, and make those 'what-ifs' of her comes true so I ca
 [sha:022d6489f9b47604441e8951872e789053eff670]{
 -- Page 177, Offset 35778.
 -- それにしても、頭がいたい。
-Still though, this headache is unbearable.
+I can't believe how much my head hurts.
 }
 [sha:f3d4e77f10c1fd4dbd45427eb63c8fcfd77a9acf]{
 -- Page 178, Offset 35779.
```

Comment:
35779, 35780, 
full hiragana

35781
full hiragana
that to link to the previous line
>but that wouldve been far too painful

35783, 
totally normal
could perhaps be shortened to link with the previous link
>Far too painful for you.

35784
totally normal

35785
full hiragana
a bit simplified
>it hurts so much that i cant stand it so　this is where i end